wintasee/hooks/openglhooks.cpp

```diff
@@ -749,15 +750,17 @@ struct OpenGLDisplayListEntry
 
 	void* AllocBuffer(int numBytes, int alignment=4)
 	{
-#if _MSC_VER > 1310
-		if(buffer)
-			_aligned_free(buffer);
-		buffer = (char*)_aligned_malloc(numBytes, alignment);
-#else
-		if(buffer)
-			free(buffer);
-		buffer = (char*)malloc(numBytes);
-#endif
+//#if _MSC_VER > 1310
+//		if(buffer)
+//			_aligned_free(buffer);
+//		buffer = (char*)_aligned_malloc(numBytes, alignment);
+//#else
+        if (buffer)
+        {
+            MemoryManager::Deallocate(buffer);
+        }
+		buffer = static_cast<char*>(MemoryManager::Allocate(numBytes, MemoryManager::ALLOC_WRITE));
+//#endif

```

Comment:
This is left there in case the _aligned_free and _aligned_malloc are important, although I think not.